source/codegen/JIT.cpp

```diff
@@ -37,9 +37,18 @@ JIT::JIT() {
 
     // Register all exported simrt functions with the JIT.
     auto exported = slang::runtime::getExportedFunctions();
+#ifdef __APPLE__
+    // Mangle names according to https://llvm.org/docs/ORCv2.html
+    auto& dl = jit->getDataLayout();
+    MangleAndInterner Mangle(jit->getExecutionSession(), dl);
```

Comment:
You probably don't need to hide this behind an ifdef for APPLE, it might just work on all platforms.

Reply:
Yes, I remove #ifdef __APPLE__ in the new commit.